src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,131 @@
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final int INDEX_ID = 1;
+    private static final int INDEX_NAME = 2;
+    private static final int INDEX_COUNTRY = 3;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest =
+                "INSERT INTO manufacturers(`manufacturers.name`, country)"
+                        + " values(?, ?);";
+        try (Connection taxiServiceConnection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturersStatement = taxiServiceConnection
+                        .prepareStatement(insertManufacturerRequest,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturersStatement.setString(INDEX_ID, manufacturer.getName());
+            createManufacturersStatement.setString(INDEX_NAME, manufacturer.getCountry());
+            createManufacturersStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturersStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(INDEX_ID, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't insert a manufacturer into DB", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String getManufacturerRequest =
+                "SELECT id, `manufacturers.name`, country FROM manufacturers"
+                        +
+                        " where id = ? and is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getManufacturerStatement =
+                        connection.prepareStatement(getManufacturerRequest)) {
+            getManufacturerStatement.setLong(INDEX_ID, id);
+            ResultSet resultSet = getManufacturerStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(createManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer from DB by id: " + id, e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        try (Connection taxiServiceConnection = ConnectionUtil.getConnection();
+                Statement getManufacturersStatement = taxiServiceConnection.createStatement()) {
+            ResultSet setOfManufacturers = getManufacturersStatement
+                    .executeQuery("SELECT * FROM taxi_service.manufacturers WHERE is_deleted = 0");
```

Comment:
```suggestion
                    .executeQuery("SELECT * FROM manufacturers WHERE is_deleted = 0");
```
немає сенсу вказувати БД. Ти її вказуєш в ConnectionUtil